inc/xbitmaps.h

```diff
@@ -339,7 +339,7 @@ static char plain_bits[] =  {   0x00, 0x00,
 int Lisp_icon_width = 88;
 int Lisp_icon_height = 58;
 
-char Lisp_icon[] = 
+const uint8_t Lisp_icon[] = 
```

Comment:
There's some extra whitespace at the end of this line that we might as well trim.